src/timelines/main.js

```diff
@@ -15,6 +16,7 @@ import {
 } from "../trials/quizTrials";
 
 const primaryTimeline = [
+  experimentStart(),
   preamble,
```

Comment:
add the `camera` trial to the timeline if video is enabled (should that check be within the `camera.js` file or here do you think?

Reply:
I don't see how it would be done in camera.js. The main timeline seems good to me.